frontend/src/components/MapView/MapView.tsx

```diff
@@ -108,7 +110,7 @@ export let MapView = withStore(
         circleOnClick={store.set('selectedProvider')}
       />}
       {selectedRepresentativePoint && <RepresentativePointPopup point={selectedRepresentativePoint} />}
-      {selectedProvider && <ProviderPopup point={selectedProvider} />}
+      {selectedProvider && <ProviderPopup store={store} point={selectedProvider} />}
```

Comment:
Do you still need this despite the withStore?

Reply:
removed in latest commit